bazel/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM launcher.gcr.io/google/ubuntu16_04
 
+ADD invocation_id_test.sh /builder/invocation_id_test.sh
```

Comment:
Why is this added to the `bazel` image?

Reply:
I started by cargo-culting and forgot to switch to taking the test script from the build source. Fixed.